index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css"
+    integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" crossorigin="anonymous">
+  <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet"
+    type="text/css" />
+  <link rel="stylesheet" href="style.css">
+  <title>WeSingAfrica Home Page</title>
+</head>
+
+<body>
+
+  <!-- Mobile Menu -->
+  <div id="wesing_menu" class="mobile-menu">
+    <ul>
+      <li class="header-link"><a href="#">Home</a></li>
+      <li class="header-link"><a href="about.html">About</a></li>
+      <li class="header-link"><a href="#project_schedule">Program</a></li>
+      <li class="header-link"><a href="./about.html#previous_events">Previous Events</a></li>
+      <li class="header-link"><a href="#wesing_guests">Guests</a></li>
+    </ul>
+    <img src="./images/Icon.png" alt="Close Menu" class="close" id="close-menu">
+  </div>
```

Comment:
- [ ] For the mobile menu, it would be excellent if the background has transparency, in this way the page still looks at the background. This makes a UX better feeling that you're always on the same page, and not on another one. 🌵 
![image](https://user-images.githubusercontent.com/20563252/128189511-91726b43-49fc-4a61-b2ca-f9e27dd87e03.png)